.github/workflows/build-with-bal-test-native.yml

```diff
@@ -17,6 +17,14 @@ on:
         default: ''
   schedule:
     -   cron: '30 18 * * *'
+    pull_request:
```

Comment:
There is an indentation issue. See the alignment of `schedule` and `pull_request`

Reply:
fixed in [b3d1391](https://github.com/ballerina-platform/module-ballerina-protobuf/pull/216/commits/b3d13911c15fd9c46d483f47f5c03c5de775b180)